website/content/en/events/2024-cloud-native-sustainability-week.md

```diff
@@ -39,6 +39,7 @@ After the successful [Sustainability Week 2023](https://tag-env-sustainability.c
 | 9 | Aarhus, Denmark | October 7th | [Link](https://community.cncf.io/events/details/cncf-aarhus-presents-cncf-sustainability-week-aarhus-2024/) | [Cloud Native Aarhus Organizers](https://community.cncf.io/aarhus/)
 | 10 | Kuala Lumpur, Malaysia | Oct. 21 | TBD | Mageshwaran Sekar, Khairul Anuar, Ahmed Zidan
 | 11 | Stuttgart, Germany | TBD| TBD | Danijel Soldo, Stefan Bergstein
+| 12 | Paris, France | Oct. 15 | TBD | Smaine Kahlouch [Cloud Native Paris](https://community.cncf.io/paris/)
```

Comment:
@Smana thanks for your PR! Can you please also update the same file for different languages? Here's an exampl from the earlier PR: https://github.com/cncf/tag-env-sustainability/pull/505/files

Thanks!🙌

Reply:
@guidemetothemoon sure! I also added the missing lines

Reply:
@Smana thank you, much appreciated!